.github/workflows/ci_test-full.yml

```
@@ -83,7 +83,7 @@ jobs:
     - name: Tests
       run: |
         # tox --sitepackages
-        coverage run --source pl_bolts -m py.test pl_bolts tests -v --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}.xml
+        coverage run --source pl_bolts -m pytest tests -v --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}.xml
```

```suggestion
        coverage run --source pl_bolts -m pytest pl_bolts tests -v --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}-${{ matrix.requires }}.xml
```
we need to keep testing package e.g. for imports